plugins/modules/grafana_datasource.py

```diff
@@ -643,11 +643,11 @@ def main():
 
     argument_spec.update(
         name=dict(required=True, type='str'),
-        state=dict(choices=['present', 'absent'],
-                   default='present'),
+        state=dict(choices=['present', 'absent'], default='present'),
         grafana_url=dict(type='str', required=True),
```

Comment:
Some options here, like `url_password` are in `plugins/module_utils/base.py` so should be removed from here

Reply:
I don't want to introduce the changes to Grafana_datasource because of the breaking change it would introduce. (url vd ds_url)